base_loadgen_experiment/code_axs.py

```diff
@@ -61,21 +65,40 @@ def parse_summary(abs_log_summary_path):
     return beautified_summary
 
 
-def parse_performance(summary, scenario_performance_map, raw=False):
+def calc_latency_cutoff_ratio(parsed_summary):
+    
+    scenario = parsed_summary["Scenario"]
+
+    if scenario == "Server":
+        return parsed_summary["99.00_percentile_latency_ns"]/parsed_summary["target_latency_ns"]
```

Comment:
Can you make `latency_cutoff_ratio` rounded off to 2 decimal places?

Reply:
sure!